libraries/apollo-debug-server/build.gradle.kts

```diff
@@ -24,9 +24,9 @@ kotlin {
 
       dependencies {
         implementation(project(":apollo-normalized-cache"))
-
-        api(project(":apollo-ast"))
-        api(project(":apollo-api"))
+        implementation(project(":apollo-normalized-cache-api"))
```

Comment:
I don't think this one should be necessary (it's `api` in `:apollo-normalized-cache`), but I'm getting red inside IJ without it 🤷‍♀️